### ISSUE TITLE
Install on Centos 7 -info

### ISSUE THREAD

**Issue by sanddweller** [pcloudcom/console-client#77]
Just leaving here for info. Mainly to document required pre-requisite packages on Centos 7.

#pre requisites for pcloud on Centos7
#
yum install -y git cmake gcc-c++ zlib libgudev1 systemd-devel kernel-devel kernel-headers fuse-devel boost boost-devel boost-system boost-program-options redhat-lsb-core boost-static
#
# download and build pcloudcc
cd /tmp
mkdir console-client
git clone https://github.com/pcloudcom/console-client.git ./console-client/
cd ./console-client/pCloudCC/
cd lib/pclsync/
make clean
make fs
cd ../mbedtls/
cmake .
make clean
make
cd ../..
cmake .
make
make install
ldconfig
cp -f pcloudcc /sbin/pcloudcc
mkdir /usr/lib/pcloud
ln -s /usr/local/lib/libpcloudcc_lib.so /usr/lib/pcloud/libpcloudcc_lib.so
pcloudcc --help
#

**Comment by ronindesign** [comment]
After following the steps above, `pcloudcc -h` was still giving error:
`pcloudcc: error while loading shared libraries: libpcloudcc_lib.so: cannot open shared object file: No such file or directory`
It appears libpcloudcc_lib.so was not being found in `/usr/lib/pcloud` created in the step:
`ln -s /usr/local/lib/libpcloudcc_lib.so /usr/lib/pcloud/libpcloudcc_lib.so`

Doing `ldd pcloudcc` (assuming you've already added the binary at `/sbin/pcloudcc`) resulted in:
```
➜  pCloudCC git:(master) ✗ ldd pcloudcc
        linux-vdso.so.1 (0x00007ffe60d9d000)
        libpcloudcc_lib.so => /tmp/console-client/pCloudCC/libpcloudcc_lib.so (0x00007f4d6283a000)
        libfuse.so.2 => /lib64/libfuse.so.2 (0x00007f4d625fc000)
        libpthread.so.0 => /lib64/libpthread.so.0 (0x00007f4d623dd000)
        libz.so.1 => /lib64/libz.so.1 (0x00007f4d621c6000)
        libdl.so.2 => /lib64/libdl.so.2 (0x00007f4d61fc2000)
        libstdc++.so.6 => /lib64/libstdc++.so.6 (0x00007f4d61c30000)
        libm.so.6 => /lib64/libm.so.6 (0x00007f4d6189c000)
        libgcc_s.so.1 => /lib64/libgcc_s.so.1 (0x00007f4d61684000)
        libc.so.6 => /lib64/libc.so.6 (0x00007f4d612c6000)
        /lib64/ld-linux-x86-64.so.2 (0x00007f4d62d88000)
```

To remedy this, I tried adding to other lib folders such as `/usr/lib64/pcloud/libpcloudcc_lib.so`, this did not work, but adding directly to the `/usr/lib64` directory did work:
`ln -s /usr/local/lib/libpcloudcc_lib.so /usr/lib64/libpcloudcc_lib.so`

Now `pcloudcc -h` works as expected.

**Comment by blackjackshellac** [comment]
It's odd that `ldd pcloudcc` is showing libpcloudcc_lib.so under /tmp.  Regardless, you can have the search include /usr/local/lib by configuring in /etc/ld.so.conf or by creating a config file in /etc/ld.so.conf.d, something like,
```
$ cat /etc/ld.so.conf.d/usr_local-x86_64.conf 
/usr/local/lib
/usr/local/lib64
```
then run `sudo ldconfig` and it should find /usr/local/lib/libpcloudcc_lib.so without the need for any symlinks under /usr/lib64.

**Comment by ronindesign** [comment]
Yeah, I think it's showing that location because I pulled the source to `/tmp/src` and was the local directory where I was running the binary from, but I'm only guessing...

Thanks for the tip on the conf, I appreciate it!
